packages/expo-stories/types/index.ts

```diff
@@ -0,0 +1,23 @@
+export interface IServerConfig {
+  projectRoot: string;
+  port: number;
+  watchRoot: string;
+}
+
+export interface IStoryManifest {
+  files: Record<string, IStoryManifestItem>;
+}
+
+export interface IStoryManifestItem {
+  id: string;
+  fullPath: string;
+  relativePath: string;
+  title: string;
+  stories: any[];
+}
+
+export interface IStoryHttpServer {
+  refreshClients: () => void;
+  start: () => void;
+  cleanup: () => void;
+}
```

Comment:
We tent to ignore all this C# or Java conventions 😅 
I don't find them useful in this scenario 😁 
```suggestion
export interface ServerConfig {
  projectRoot: string;
  port: number;
  watchRoot: string;
}

export interface StoryManifest {
  files: Record<string, StoryManifestItem>;
}

export interface StoryManifestItem {
  id: string;
  fullPath: string;
  relativePath: string;
  title: string;
  stories: any[];
}

export interface StoryHttpServer {
  refreshClients: () => void;
  start: () => void;
  cleanup: () => void;
}
```